app/views/ctc/questions/filed_prior_tax_year/_pr_edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<% @main_question = t("views.ctc.questions.filed_prior_tax_year.pr.title", prior_tax_year: prior_tax_year) %>
```

Comment:
interesting. i think i was expecting the approach of creating a separate controller / new question for the different question and then using the `show?` function to decide whether to show the PR one or the US one. Tbh, this is actually simpler to reason about from within the controller.

Reply:
this one! but also, where is the controller change for this? dont we need to have logic to render this when its PR?